database/insertData/03_template_D_basicReviewTesting.js

```diff
@@ -266,29 +266,61 @@ exports.queries = [
           }
```

Comment:
We're going to be ditching most of applications and just leaving the templates in `insertData` -- actual application data will be handled by Snapshots (#287) soon, so rather than have to re-define all the element links in the basicReviewTesting applications now, let's wait till after that and then just delete the ids from this template.